depends/patches/qt/qfixed-coretext.patch

```diff
@@ -1,34 +0,0 @@
-From dbdd5f0ffbce52c8b789ed09f1aa3f1da6c02e23 Mon Sep 17 00:00:00 2001
```

Comment:
Are you sure this patch is no longer required?
Compiling 99d2d3c I'm seeing:
```
compiling qtestcase.cpp
mac/qfontengine_coretext.mm:833:20: error: qualified reference to 'QFixed' is a constructor name rather than a type in this context
    return QFixed::QFixed(int(CTFontGetUnitsPerEm(ctfont)));
                   ^
1 error generated.
```